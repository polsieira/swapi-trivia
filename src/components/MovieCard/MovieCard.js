import React from 'react';

const MovieCard = () => {
  <div className="movie-card">
    <h3>Title</h3>
    <p>Episode</p>
    <p>Year</p>
    <button>View Characters</button>
  </div>
}

export default MovieCard;