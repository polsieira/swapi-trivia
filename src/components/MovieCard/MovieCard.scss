$star-wars-yellow: #FFE81F;

.movie-cards {
  width: 190px;
  height: 500px;
}

.movie-card-1 {
 background-image: url('../../images/phantom-menace.jpg');
 background-repeat: no-repeat;
 background-size: auto;
 background-position: center;
 border: red solid 5px;
}

.movie-card-2 {
  background-image: url('../../images/clones.jpg');
  background-repeat: no-repeat;
  background-size: auto;
  background-position: center;
  border: green solid 5px;
}

.movie-card-3 {
  background-image: url('../../images/revenge-of-the-sith.jpg');
  background-position: left;
  background-repeat: no-repeat;
  background-size: auto;
  border: orange solid 5px;
}

.movie-card-4 {
  background-image: url('../../images/new-hope.jpg');
  background-position: center;
  background-repeat: no-repeat;
  background-size: auto;
  border: white solid 5px;
}

.movie-card-5 {
  background-image: url('../../images/empire-strikes-back.jpg');
  background-position: center;
  background-repeat: no-repeat;
  background-size: auto 100%;
  border: blue solid 5px;
}

.movie-card-6 {
  background-color: red;
}

.movie-card-7 {
  background-color: yellow;
}