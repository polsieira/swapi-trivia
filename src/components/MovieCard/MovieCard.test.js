import React from 'react';
import { shallow } from 'enzyme';

import MovieCard from './MovieCard';

describe('MovieCard', () => {
  it('should match the snapshot with all data passed correctly', () => {
    const wrapper = shallow(<MovieCard 
      id="1"
      title="A New Hope"
      episode="4"
      year = "A New Hope"
      />)

    expect(wrapper).toMatchSnapshot();
  });


});