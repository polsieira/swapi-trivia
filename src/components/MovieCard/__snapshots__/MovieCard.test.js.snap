// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard should match the snapshot with all data passed correctly 1`] = `ShallowWrapper {}`;
